### PR TITLE
Show ToastUtils instead of using Toast.makeText directly

### PR DESCRIPTION
**Pull request by oguzkocer**
I realized we were inconsistent with showing the toast, so I thought this PR could help. I specifically didn't make any changes outside of the main repo and kept the usage of `Toast.makeText` where it made sense (custom toasts).

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java [hunk 1]
```diff
@@ -194,7 +193,7 @@ public void onErrorResponse(VolleyError response) {
                             }
                             AppLog.d(T.THEMES, getString(R.string.theme_auth_error_authenticate));
                         } else {
-                            Toast.makeText(ThemeBrowserActivity.this, R.string.theme_fetch_failed, Toast.LENGTH_LONG)
+                            ToastUtils.showToast(ThemeBrowserActivity.this, R.string.theme_fetch_failed, ToastUtils.Duration.LONG)
                                     .show();
```
Comment (tonyr59h) [inline WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java]: This `.show()` call should be removed ;)

Comment (oguzkocer) [inline WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java]: Even after reviewing the PR twice, I still missed one 👎 Fixed in 8b3acc86f7d7d58fae640229e3f2024672e40fb7

**Comment by oguzkocer** [comment]
@tonyr59h Thanks for picking this up. Any ideas how to test this PR without going through every single page? I feel like PR review will have to be enough for this one.

**Comment by tonyr59h** [comment]
> I feel like PR review will have to be enough for this one.

Agreed, also a Clean->Build for good measure :)

:shipit:

**Pull request merged**